docs/images.md

```diff
@@ -218,7 +218,7 @@ Note that you must specify some width and height style attributes.
 
 ## iOS Border Radius Styles
 
-Please note that the following corner specific, border radius style properties are currently ignored by iOS's image component:
+Please note that the following corner specific, border radius style properties are may ignored by iOS's image component:
```

Comment:
```suggestion
Please note that the following corner specific, border radius style properties might be ignored by iOS's image component:
```

Reply:
I'm going to merge the suggestion and land the PR, so it won'y become stale. 😉 

Also the duplicates start to appear:
* https://github.com/facebook/react-native-website/pull/2908